entries/u/unisuper.com.au.json

```
@@ -2,14 +2,16 @@
   "UniSuper": {
     "domain": "unisuper.com.au",
     "url": "https://www.unisuper.com.au/",
-    "contact": {
-      "twitter": "UniSuperNews"
-    },
+    "tfa": [
+      "sms",
+      "email"
+    ],
+    "documentation": "https://www.unisuper.com.au/contact-us/accessing-online-accounts",
```

Since the document doesn't mention 2FA set up but instead recovering from different types of authentication issues, I suggest you use the recovery attribute instead:
```suggestion
    "recovery": "https://www.unisuper.com.au/contact-us/accessing-online-accounts",
```